php/7.4/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM php:7.4-fpm-alpine
+
+RUN apk upgrade --update \
+    && apk --update add --no-cache --virtual .phpize-deps $PHPIZE_DEPS imagemagick-dev libtool \
+    && pecl install imagick \
+    && docker-php-ext-configure exif \
+    && docker-php-ext-install -j$(nproc) mysqli \
+    && docker-php-ext-install -j$(nproc) exif \
+    && docker-php-ext-install -j$(nproc) pdo pdo_mysql \
+    && docker-php-ext-enable imagick \
+    && docker-php-ext-enable exif \
+    && apk add --no-cache --virtual .imagick-runtime-deps imagemagick \
```

Comment:
Also I don't see the point of using virtual package here or is there a reason I'm not aware of?

Reply:
It turned out that from the aspect of `imagemagick` it's not relevant to use virtual package, but it had a reason for `imagemagick-runtime-deps`, the new commit provides a new way of handling dependencies and the cleanup